internal/proxy/options.go

```diff
@@ -215,10 +220,19 @@ func parseProviderInfo(o *Options) error {
 		return errors.New("provider-url must include scheme and host")
 	}
 
+	proxyproviderURL, err := url.Parse(o.ProxyProviderURLString)
```

Comment:
nitpick here, but should maybe be cased as `proxyProviderURL`

Reply:
changed it 👍